test/e2e/rescheduling_test.go

```diff
@@ -145,3 +145,182 @@ var _ = ginkgo.Describe("reschedule testing", func() {
+				policyName = deploymentNamePrefix + rand.String(RandomStrLength)
+				deploymentNamespace = testNamespace
+				deploymentName = policyName
+				deployment = testhelper.NewDeployment(deploymentNamespace, deploymentName)
+				deployment.Spec.Replicas = pointer.Int32Ptr(1)
+				// set ReplicaSchedulingType=Duplicated.
+				policy = testhelper.NewPropagationPolicy(policyNamespace, policyName, []policyv1alpha1.ResourceSelector{
+					{
+						APIVersion: deployment.APIVersion,
+						Kind:       deployment.Kind,
+						Name:       deployment.Name,
+					},
+				}, policyv1alpha1.Placement{
+					ReplicaScheduling: &policyv1alpha1.ReplicaSchedulingStrategy{
+						ReplicaSchedulingType: policyv1alpha1.ReplicaSchedulingTypeDuplicated,
+					},
+				})
+			})
+
+			ginkgo.It("when the ReplicaSchedulingType of the policy is Duplicated, reschedule testing", func() {
+				ginkgo.By("create deployment and policy")
+				framework.CreatePropagationPolicy(karmadaClient, policy)
+				framework.CreateDeployment(kubeClient, deployment)
+				ginkgo.DeferCleanup(func() {
+					framework.RemoveDeployment(kubeClient, deployment.Namespace, deployment.Name)
+					framework.RemovePropagationPolicy(karmadaClient, policy.Namespace, policy.Name)
+				})
+
+				ginkgo.By(fmt.Sprintf("Joinning cluster: %s", newClusterName))
+				karmadaConfig := karmadactl.NewKarmadaConfig(clientcmd.NewDefaultPathOptions())
+				opts := karmadactl.CommandJoinOption{
+					DryRun:            false,
+					ClusterNamespace:  "karmada-cluster",
+					ClusterName:       newClusterName,
+					ClusterContext:    clusterContext,
+					ClusterKubeConfig: kubeConfigPath,
+				}
+				err := karmadactl.RunJoin(karmadaConfig, opts)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+				// wait for the current cluster status changing to true
+				framework.WaitClusterFitWith(controlPlaneClient, newClusterName, func(cluster *clusterv1alpha1.Cluster) bool {
+					return meta.IsStatusConditionPresentAndEqual(cluster.Status.Conditions, clusterv1alpha1.ClusterConditionReady, metav1.ConditionTrue)
+				})
+
+				ginkgo.By("check whether the deployment is rescheduled to a new cluster")
+				gomega.Eventually(func(g gomega.Gomega) bool {
+					targetClusterNames := framework.ExtractTargetClustersFrom(controlPlaneClient, deployment)
+					return !testhelper.IsExclude(newClusterName, targetClusterNames)
+				}, pollTimeout, pollInterval).Should(gomega.BeTrue())
+			})
+		})
+
+		ginkgo.Context("testing clusterAffinity of the policy", func() {
+			ginkgo.BeforeEach(func() {
+				initClusterNames = []string{"member1", "member2", newClusterName}
+				policyNamespace = testNamespace
+				policyName = deploymentNamePrefix + rand.String(RandomStrLength)
+				deploymentNamespace = testNamespace
+				deploymentName = policyName
+				deployment = testhelper.NewDeployment(deploymentNamespace, deploymentName)
+				deployment.Spec.Replicas = pointer.Int32Ptr(1)
+				// set Placement to empty.
```

Comment:
What do you mean `set to empty`?

Reply:
https://github.com/karmada-io/karmada/issues/2261

<img width="780" alt="image" src="https://user-images.githubusercontent.com/17962021/186571014-efcbcb66-944f-4b80-b483-040a4b144961.png">

Reply:
Sorry, I made a mistake